apps/docs/src/app/core/component-docs/feed-list-item/examples/fli-action/fli-action-example.component.html

```diff
@@ -28,31 +28,36 @@
         ariaLabel="Feed list item example"
     >
         <fd-feed-list-action>
-            <button
-                aria-label="action"
-                fd-button
-                fdType="transparent"
-                [compact]="true"
-                [fdMenuTrigger]="menu"
-                glyph="menu"
-            ></button>
-            <fd-menu #menu>
-                <li fd-menu-item>
-                    <a href="#" fd-menu-interactive>
-                        <span fd-menu-title>Option 1</span>
-                    </a>
-                </li>
-                <li fd-menu-item>
-                    <a href="#" fd-menu-interactive>
-                        <span fd-menu-title>Option 2</span>
-                    </a>
-                </li>
-                <li fd-menu-item>
-                    <a href="#" fd-menu-interactive>
-                        <span fd-menu-title>Option 3</span>
-                    </a>
-                </li>
-            </fd-menu>
+            <fd-action-sheet [compact]="true" #actionSheet>
+                <fd-action-sheet-control>
+                    <button fd-button [compact]="true" glyph="overflow" fdType="transparent"></button>
```

Comment:
please add `title="More"` in order to make a11y team happy

Reply:
done